Metaverse/Source/Metaverse/Private/Components/WalletConnectTriggerComponent.cpp

```diff
@@ -145,6 +146,33 @@ void UWalletConnectTriggerComponent::Connect() {
   }
 }
 
+
+
+//
+// Disconnect
+//
+void UWalletConnectTriggerComponent::Disconnect() {
+  _PlayCppSdk =
+      (APlayCppSdkActor *)UGameplayStatics::GetActorOfClass(
+          GetWorld(), APlayCppSdkActor::StaticClass());
+  if (_PlayCppSdk) {
+  } else {
+    _PlayCppSdk =
+        (APlayCppSdkActor *)GetWorld()->SpawnActor(
+            APlayCppSdkActor::StaticClass());
+  }
+
+  bool success;
```

Comment:
where is success is set ?

Reply:
It would be passed into the ClearSession function and set inside the function.